docs/book/validators/phone-number.md

```diff
@@ -0,0 +1,143 @@
+# PhoneNumber
+
+`Laminas\I18n\PhoneNumber\Validator\PhoneNumber` allows you to determine if a given value **is a valid phone number**.
+Phone number formats are country specific.
+
+## Basic Usage
+
+```php
+use Laminas\I18n\PhoneNumber\Validator\PhoneNumber;
+
+$validator = new PhoneNumber();
+$validator->isValid('+4930123456'); // true
+```
```

Comment:
The most code example can be reduced.

```suggestion
```php
$validator = new Laminas\I18n\PhoneNumber\Validator\PhoneNumber();
$validator->isValid('+4930123456'); // true
```
```